src/main/java/it/gov/pagopa/pu/pagopapayments/connector/pu_sil/client/PuSilClient.java

```diff
@@ -0,0 +1,21 @@
+package it.gov.pagopa.pu.pagopapayments.connector.pu_sil.client;
+
+import it.gov.pagopa.pu.pagopapayments.connector.pu_sil.config.PuSilApisHolder;
+import it.gov.pagopa.pu.pusil.dto.generated.AmountUpdatesDTO;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class PuSilClient {
+
+  private final PuSilApisHolder apisHolder;
+
+  public PuSilClient(PuSilApisHolder apisHolder) {
+    this.apisHolder = apisHolder;
+  }
+
+  public AmountUpdatesDTO getAmountUpdates(Long orgSilServiceId, String nav, String accessToken) {
+      return apisHolder.getAmountUpdatesApi(accessToken).getAmountUpdates(orgSilServiceId, nav);
```

Comment:
Let's intercept the Conflict error and throw NotPayableSilActualizedAmountException